netlogo-gui/src/main/properties/ReporterLineEditor.scala

```diff
@@ -18,6 +18,11 @@ abstract class ReporterLineEditor(accessor: PropertyAccessor[String],
     editor,
     ScrollPaneConstants.VERTICAL_SCROLLBAR_NEVER,
     ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER)
-  override def get = super.get.map(_.trim).filter(_.nonEmpty)
-  override def getMinimumSize = new Dimension(500, 500)
+  override def get = super.get.map(_.trim)
```

Comment:
would keeping the .nonEmpty filter cause an error? If so a comment might be helpful